controller/src/main/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessors.java

```diff
@@ -326,6 +328,7 @@ protected void shutDown() {
 
         String scope = config.getScopeName();
         CompletableFuture<Void> future = createScope(scope);
+
```

Comment:
nit: Remove redundant line changes.

Reply:
done :)

Reply:
The redundant line change still shows as a diff.